pre-commit/templates/connector_detail.md

```diff
@@ -0,0 +1,92 @@
+[comment]: # "Auto-generated SOAR connector documentation"
+# {{connector.name}}
+
+Publisher: {{connector.publisher}}  
+Connector Version: {{connector.app_version}}  
+Product Vendor: {{connector.product_vendor}}  
+Product Name: {{connector.product_name}}  
+Product Version Supported (regex): "{{connector.product_version_regex}}"  
+Minimum Product Version: {{connector.min_phantom_version}}  
+
+{{connector.description}}
+
+{%- if connector.author_notes %}
+
+{{connector.author_notes | string}}
+
+{%- endif %}
+
+{%- if connector.configuration %}
+
+### Configuration Variables
+The below configuration variables are required for this Connector to operate.  These variables are specified when configuring a {{connector.product_name}} asset in SOAR.
+
+VARIABLE | REQUIRED | TYPE | DESCRIPTION
+-------- | -------- | ---- | -----------
+{%- for name, parameters in connector.configuration.items() %}
+{%- if name != "ph" %}
+**{{name}}** | {% if parameters.required == True %} required {% else %} optional {% endif %} | {{ parameters.data_type }} | {{ parameters.description }}
+{%- endif %}
+{%- endfor %}
+{%- endif %}
+
+{%- if connector.app_config %} <!-- TODO: connector.app_config might be deprecated. Remove? -->
+
+### (Deprecated?) App Config Variables
```

Comment:
I *think* I should remove this section, since App Config Variables seem largely unused today. Would like other thoughts on it.

Reply:
agreed, I've never seen this used in recent memory. which seems further confirmed by Tony's response on slack

Reply:
Sounds good. Updated. Fixing tests now.

Reply:
Tests fixed.